charts/openebs/openebs-target-container-failure/openebs-target-container-failure.chartserviceversion.yaml

```diff
@@ -1,29 +1,32 @@
 apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
-  name: openebs-target-failure
-  version: 0.1.2
+  name: openebs-target-container-failure
+  version: 0.1.1
   annotations:
     categories: OpenEBS
     vendor: CNCF
     repository: https://github.com/litmuschaos/chaos-charts
     support: https://slack.openebs.io/
 spec:
-  displayName: openebs-target-failure 
+  displayName: openebs-target-container-failure 
   categoryDescription: >
      Kill the cstor target/Jiva controller container and check if gets created again
   keywords:
     - Kubernetes
     - Storage Controller
     - OpenEBS
+  platforms:
+    - GKE
+    - OpenShift
   maturity: alpha
   maintainers:
     - name: shubham chaudhary
       email: shubham.chaudhary@mayadata.io
   minKubeVersion: 1.12.0
   provider:
     name: Mayadata
-  readme: "https://github.com/mayadata-io/litmus/blob/master/experiments/chaos/openebs_target_failure/README.md"
+  readme: "https://github.com/litmuschaos/litmus/blob/master/experiments/openebs/openebs_target_container_failure/README.md"
```

Comment:
The link should be: `https://github.com/litmuschaos/litmus/blob/master/experiments/openebs/openebs-target-container-failure/README.md`